src/main.rs

```diff
@@ -6,7 +6,7 @@ use std::net::TcpListener;
 use std::net::TcpStream;
 
 const DEFAULT_IP: &str = "127.0.0.1";
-const DEFAULT_PORT: &str = "7878";
+const DEFAULT_PORT: &str = "8080";
```

Comment:
@vuong-buihv any specific reason for port 7878? Or just randomly chosen :p
Port 80 (or 8080) is most commonly used for HTTP

Reply:
Just randomly chosen 😆